src/components/NFTView.vue

```diff
@@ -169,7 +171,7 @@ function handleFallbackNft(event) {
               <span
                 class="text-gray-100 text-xs overflow-hidden whitespace-nowrap text-ellipsis"
                 :title="nft.collectionName"
-                >{{ nft.collectionName }}</span
+                >{{ nft.identifier }}</span
```

Comment:
And what happens in EVM where we have collectionName?

Reply:
fixing this

Reply:
[done](https://github.com/arcana-network/wallet-ui/pull/615/commits/34dea73d124f4fbfe4a26c2a34a578c3e1158943)